projects/configs/tag-operations/base.yaml

```diff
@@ -0,0 +1,660 @@
+  group: 0x0010
+  element: 0x0010
+  op: "replace"
+#https://dicom.innolitics.com/ciods/ultrasound-image/patient/00100010
+#https://dicom.innolitics.com/ciods/ct-image/patient/00100010
+#https://dicom.innolitics.com/ciods/ultrasound-multi-frame-image/patient/00100010
+#https://dicom.innolitics.com/ciods/x-ray-radiofluoroscopic-image/patient/00100010
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/patient/00100010
+#https://dicom.innolitics.com/ciods/rt-image/patient/00100010
+#https://dicom.innolitics.com/ciods/digital-x-ray-image/patient/00100010
+- name: "Patient ID"
+  group: 0x0010
+  element: 0x0020
+  op: "secure-hash"
+- name: "Patient's Birth Date"
+  group: 0x0010
+  element: 0x0030
+  op: "replace"
+- name: "Patient's Sex"
+  group: 0x0010
+  element: 0x0040
+  op: "keep"
+#################################### 0018 Group ###################################
+#
+#
+- name: "Scan Options Attribute"
+  group: 0x0018
+  element: 0x0022
+  op: "replace"
+#https://dicom.innolitics.com/ciods/ct-image/ct-image/00180022
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180022
+#https://dicom.innolitics.com/ciods/x-ray-radiofluoroscopic-image/x-ray-image/00180022
+#https://dicom.innolitics.com/ciods/x-ray-angiographic-image/x-ray-image/00180022
+- name: "MR Acquisition Type"
+  group: 0x0018
+  element: 0x0023
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180023
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/mr-pulse-sequence/00180023
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/mr-pulse-sequence/00180023
+- name: "Repetition Time"
+  group: 0x0018
+  element: 0x0080
+  op: "keep"
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009229/00189112/00180080
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189112/00180080
+- name: "Number of Averages"
+  group: 0x0018
+  element: 0x0083
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180083
+#https://dicom.innolitics.com/ciods/mr-spectroscopy/mr-spectroscopy-multi-frame-functional-groups/52009229/00189119/00180083
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image-multi-frame-functional-groups/52009230/00189119/00180083
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-color-image-multi-frame-functional-groups/52009229/00189119/00180083
+- name: "Magnetic Field Strength"
+  group: 0x0018
+  element: 0x0087
+  op: "keep"
+#https://dicom.innolitics.com/ciods/mr-image/mr-image/00180087
+#https://dicom.innolitics.com/ciods/enhanced-mr-image/enhanced-mr-image/00180087
+#https://dicom.innolitics.com/ciods/enhanced-mr-color-image/enhanced-mr-image/00180087
+#https://dicom.innolitics.com/ciods/legacy-converted-enhanced-mr-image/enhanced-mr-image/00180087
+- name: "Spacing Between Slices"
+  group: 0x0018
+  element: 0x0088
+  op: "keep"
```

Comment:
These look like they should all be in MRI rather than base?

Reply:
[acc3b3c](https://github.com/SAFEHR-data/PIXL/pull/498/commits/acc3b3cebbbbb8987bc01783ccbc89fee30d51bf)